src/app/public/modules/format/app-format.ts

```diff
@@ -2,7 +2,9 @@ import {
   Injectable
 } from '@angular/core';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root'
+})
 export class SkyAppFormat {
```

Comment:
I'll admit I don't know much about this service but I noticed this wasn't in any modules before - how is this being consumed today?

Reply:
It's not being used by any of our component libraries, but since it's in the public API we probably need to keep it around. Before this change, consumers would have to add this service to AppExtrasModule providers.